lib/output/utils.js

```diff
@@ -36,15 +47,22 @@ function initCtorRegistry(globalObject) {
     ctorRegistry[`%${intrinsic}%`] = globalObject[intrinsic];
   }
 
-  // TODO: Also capture `%AsyncIteratorPrototype%`
   ctorRegistry["%IteratorPrototype%"] = Object.getPrototypeOf(
     Object.getPrototypeOf(new ctorRegistry["%Array%"]()[Symbol.iterator]())
   );
+  ctorRegistry["%AsyncIteratorPrototype%"] = AsyncIteratorPrototype;
```

Comment:
Add a comment here saying that this is not actually correct, and integrators should use registerConstructor to do the right thing.